migrate/v0_15/incentive.go

```diff
@@ -71,19 +75,21 @@ func Incentive(incentiveGS v0_14incentive.GenesisState) v0_15incentive.GenesisSt
 	)
 
```

Comment:
What is the idea for this TODO? Is the thought to set previous accumulation time to genesis time, in case the export is missing one of the genesis accumulation times?

Reply:
It was for just usdx rewards. When the params were removed on kava-7 the previous accumulation time stopped updating. So if rewards are ever enabled again, the begin blocker could try and accumulate rewards since params were removed.
It's not really necessary as the accumulators take into account the start time. It would only be a problem if rewards were added with a really old start time.